service/iot-service-client/src/main/java/com/microsoft/azure/sdk/iot/service/digitaltwin/generated/implementation/DigitalTwinsImpl.java

```diff
@@ -210,8 +200,7 @@ public Void call(ServiceResponseWithHeaders<Void, DigitalTwinUpdateDigitalTwinHe
         }
         Validator.validate(digitalTwinPatch);
         final String apiVersion = "2020-09-30";
-        final String ifMatch = null;
```

Comment:
This one worries me a bit. The change is fine, but please follow up with the PnP swagger folks as to why the if match header can't be set here. This seems like that issue we had in ADT land where the if-none-match header was always set to "*" because it only had one value defined in the swagger